debian/postinst

```diff
@@ -89,10 +89,11 @@ if [ "$1" = configure ]; then
 
     systemctl --quiet is-enabled crowdsec || systemctl unmask crowdsec && systemctl enable crowdsec
 
-    if [ -z "$(ss -nlt 'sport = 8080' | grep -v ^State)" ] || [ "$LAPI" = false ]; then
+    PORT=$(cat /etc/crowdsec/config.yaml|grep listen_uri |cut -d ":" -f3)
```

Comment:
why not using `cscli config show` to achieve this ? less error prone I'd guess (+ we will want the same for RPMs)

Reply:
yes, you're right, let's do this.